aict_tools/configuration.py

```diff
@@ -108,30 +104,32 @@ def from_yaml(cls, configuration_path):
             return cls(yaml.load(f))
 
     def __init__(self, config):
-        self.has_multiple_telescopes = config.get('multiple_telescopes', False)
-        self.runs_key = config.get('runs_key', 'runs')
-        self.true_energy_column = config.get('true_energy_column')
+        self.data_format = config.get('data_format', "simple")
+        self.telescopes = config.get('telescopes', None)
         self.size_column = config.get('size')
-
-        if self.has_multiple_telescopes:
-            self.telescope_events_key = config.get('telescope_events_key', 'events')
-            self.array_events_key = config.get('array_events_key', 'array_events')
-
-            self.array_event_id_column = config.get(
-                'array_event_id_column', 'array_event_id'
-            )
-            self.run_id_column = config.get('run_id_column', 'run_id')
-        else:
-            self.telescope_events_key = config.get('telescope_events_key', 'events')
-            self.run_id_column = config.get('run_id_column', 'run_id')
-
-            self.array_events_key = None
-            self.array_event_id_column = None
-
         self.energy_unit = u.Unit(config.get('energy_unit', 'GeV'))
         self.seed = config.get('seed', 0)
         np.random.seed(self.seed)
 
+        self.true_energy_column = config.get('true_energy_column')
```

Comment:
I think also the true energy column should be fixed in the CTA dataformat.

Probably it would be best, just to make a list of non-cta config options and check if any of them are in the config.

```
SIMPLE_OPTIONS = set(...)

if data_format == "CTA" and SIMPLE_OPTIONS.intersection(config.keys()):
   ...
```

Also, it would probably make the code a lot cleaner, if we have two methods like this:

```
def __init__(self, config):
    if config['data_format'] == 'CTA':
        self.parse_cta(config)
    else:
        self.parse_simple(config)
```
```

Reply:
> Probably it would be best, just to make a list of non-cta config options and check if any of them are in the config.
> 
> ```
> SIMPLE_OPTIONS = set(...)
> 
> if data_format == "CTA" and SIMPLE_OPTIONS.intersection(config.keys()):
>    ...
> ```

Do you mean to use this to throw an exception or to override default CTA values? Because in the latter case I struggle with the use of `__slots__`. But maybe we dont want to support that anyway right now?

Reply:
No, this should throw an error. We want to only support the fixed CTA DL1 format like it is now written by the stage1 tool

Reply:
>  __slots__. But maybe we dont want to support that anyway right now?

`__slots__` are not really required here, it's just nice to have so typos of attributes raises exceptions. If you can better due without `__slots__`, get rid of them.